ports/nrf/boards/wrl13990/README.md

```diff
@@ -0,0 +1,68 @@
+# WRL-13990 Board Support
+
+The Sparkfun `WRL-13990` is a very basic, no thrills, `nRF52832-QFAA` based board. It can be used as a DIL socket and eases the development of Bluetooth Low Energy or other ARM Cortex-M4 applications.
+
+## Board Pins
+An overview of all available pins is in the table below. Most of the pins can be used for PWM, UART, SPI, I2C, etc. However some pins are special like the ADC and NFC provision. Please refer to the datasheet for additional information.
+
+- When using NFC pins take special care about configuration. See note in [datasheet](http://infocenter.nordicsemi.com/pdf/nRF52832_PS_v1.4.pdf), page 17..18. At reset the pins are set to NFC mode!
+- When UART hardware handshaking is required, select the GPIO's you like and set them accordingly in the `mpconfigboard.h`.
+
+
+BOARD | CPU | REMARK
```

Comment:
A minor syntactic sugar detail on this table as well. Could you expand the columns so that the PIPE characters get aligned? It makes it a bit easier to read when using a text editor.

Reply:
Ok that should do it :-)